crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -106,6 +106,10 @@ where
         })
     }
 
+    pub fn inner(&self) -> &RocksDb<Historical<Description>> {
```

Comment:
The problem with exposing it is that anyone can commit to the internal database bypassing the historical functionality. Maybe it is possible to just add backup functionality directly?

Reply:
Or maybe we can put it under the "test-helpers" feature flag?

Reply:
Maybe we can add a read-only rapper for this somehow?

Reply:
draft pr: https://github.com/FuelLabs/fuel-core/pull/2535